src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result += days * 40;
+  } else if (days < 7) {
+    result += days * 40 - 20;
+  } else {
+    result += days * 40 - 50;
+  }
```

Comment:
create vars for the price and discounts
it is easier to change values than to search for numbers